sdk/typescript/src/cryptography/ed25519-keypair.ts

```diff
@@ -6,6 +6,15 @@ import { Base64DataBuffer } from '../serialization/base64';
 import { Keypair } from './keypair';
 import { Ed25519PublicKey } from './ed25519-publickey';
 import { SignatureScheme } from './publickey';
+import {
+  isValidHardenedPath,
+  mnemonicToSeedHex,
+  normalizeMnemonics,
+  validateMnemonics,
+} from './mnemonics';
+import { derivePath, getPublicKey } from '../utils/ed25519-hd-key';
+
+export const DEFAULT_ED25519_DERIVATION_PATH = "m/44'/784'/0'/0'/0'";
```

Comment:
Can we make a templatized version of this that we can pass account indexes into? Would be useful for consistent multi-account setups.

Reply:
different wallet actually has different ways of using the BIP44 path https://gist.github.com/miguelmota/034c81886bd23b25c64ec5eda9251641 for example ledger legacy vs ledger legacy uses the account_index vs the address_index derives account differently. 

i would prefer to keep the full path as a param at the minimum. when we provide multi-account setup, I agree we can have a more restrictive API just to pass in the acocunt_index.

Reply:
@Jordan-Mysten's comment makes sense, we can merge now as we don't still support multi-accounts, and @joyqvq please create an issue to revisit when we eventually handle multi-accounts.